JAVASCRIPT.md

```diff
@@ -276,6 +277,10 @@
   * enforce the consistent use of double quotes in JSX attributes
   [read more](https://eslint.org/docs/rules/jsx-quotes)
 
+## jsx-one-expression-per-line
+  * limits every line in JSX to one expression each.
```

Comment:
do we want to add this to https://github.com/ForwardFinancing/code_styles/blob/master/eslintrc.json so we lint with it automatically?

Reply:
nope that is all 🎆